tests/org.eclipse.jface.tests/src/org/eclipse/jface/tests/viewers/TableViewerWithLimitTest.java

```diff
@@ -0,0 +1,251 @@
+package org.eclipse.jface.tests.viewers;
```

Comment:
File header missing.

Please also make sure all new tests are added to existing testsuite